### ISSUE TITLE
[Low] Import label is read twice by Windows screen readers

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/wai-atag-report-tool#75]
There is a text node saying "Import" that is read by JAWS and NVDA in addition to the label for the button "import". Given that the button already has its own label, I think the node should be hidden for screen readers to avoid unnecessary repetition.

**Comment by hidde** [comment]
Thanks for reporting this. 

I have investigated this a bit, but I'm not sure how to resolve this, because in the markup there is only one label.

The markup:

```html
<input type="file" id="import-evaluation" class="visuallyhidden" accept="application/json"/>
<label for="import-evaluation" class="button button-secondary">Import evaluation</label>
```

What I'm doing is cheating… it is a file upload input with a label, the input is visually hidden, the label is styled as a button, so that it behaves like one. 

In the most recent version I have updated it to say “Import evaluation”, just wondering if you still hear this twice? I am curious where NVDA/JAWS are getting the second label from.

**Comment by daniel-montalvo** [comment]
I tried at https://atag-report-tool.hdv.now.sh/your-evaluationItThey both continue to say it twice. There seems to be no easy way for me to fix this. I have been playing with:

* Setting `aria-hidden="true"` in the label (which hides the whole text in Firefox).
* Code the `label` text as a `paragraph` text, add and id to it, style it with the classes you had previously, and then use `aria-labelledby` to reference it from the input, which does not work in Firefox either as they use the Browse string when they encounter a input type="file".

I think the cure is worse than the decease here. It is good to have this documented but I wonder if it deserves more time than what we have already invested.

**Comment by hidde** [comment]
Thanks for checking this. It seems like in most browsers a file input + label creates two entries in the accessibility tree, with both a label.

This probably makes sense when both are visible, but in our case, the input is actually rendered invisible.

Something else I have played with:

* Wrap `input` inside `label`. This does fix the double label issue, but now I'll need the `focus-within` pseudo class for focus styles and that doesn't have very broad browser support (does not work in IE11)

I'll leave this open for a bit in case we get new ideas to make this better.

**Comment by hidde** [comment]
Thanks again for reporting this Daniel. Closing the issue as I see no away to fix it right now. 

As you say, cure sems worse than decease, let's have it in the archives for documentation.
